qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -130,6 +130,29 @@
        </ul>
      </description>
    </release>
+   <release version="2022.09.22-3" date="2022-09-22">
+     <description>
+       <ul>
+         <li>New OSM maps data as of October 29</li>
+         <li></li>
```

Comment:
• New OSM data as of October 19
• Fixed search indicator in Viewport mode
• Fixed search table update, now old results are always replaced by new ones
• Minor performance improvements
• Dump corrupted KMLs to the log on startup

Routing and Navigation:
• Show "Roads to avoid" options and warnings on the route also for walking, cycling, and metro routes
• Fixed some routing issues, including wrong routing via highway exits/links
• Enable Catalan TTS
• Show speed camera on tap

Search:
• Show all found results on the map screen
• Improved villages search
• Show only matched categories if they were selected from the list, not names
• Fixed street synonyms for nordic languages

Styles:
• Use regional metro/subway icons for entrances and stations
• Added brewery, plumber, tailor, carpenter, sawmill, beekeeper, blacksmith, electrician, painter, and other craft icons
• Added natural=shrubbery, natural=desert, leisure=dance, amenity=charging_station-motorcar, and more barrier types
• Draw international airport icon on z7-9
• Draw metro for major cities starting from z12
• Do not draw railway station icon for building=train_station
• Updated icons for garden, park, forest, national park, nature reserve, power, chimney
• Add historic battlefield, gallows, pillory, memorial cross, stolperstein and war_memorial
• Fixed historic=* priority
• Support sauna=yes tags
• Better colors for natural=scrub
• Draw entrance icon instead of a dot

Translations:
• Use larger, more geographically significant names for Oregon maps in Downloader
• Updated Catalan, Chinese, Dutch, Finnish, German, Hungarian, Italian, Polish, Portuguese (European and Brazilian), Turkish translations

Reply:
Done